sdk/src/client/node_api/core/routes.rs

```diff
@@ -147,8 +147,8 @@ impl Client {
     ) -> Result<ValidatorsResponse> {
         const PATH: &str = "api/core/v3/validators";
         let query = query_tuples_to_query_string([
-            page_size.into().map(|i| ("pageSize", i.to_string())),
-            cursor.into().map(|i| ("cursor", i)),
+            page_size.into().map(|page_size| ("pageSize", page_size.to_string())),
+            cursor.into().map(|cursor| ("cursor", cursor)),
```

Comment:
Don't you think we should automatically request the next pages and merge them together like we did for the outputs https://github.com/iotaledger/iota-sdk/blob/d9eea41a7b86ad1b4c3bd39e8f9ac5a40b70d5e8/sdk/src/client/node_api/indexer/mod.rs#L34?

Reply:
seems like something we can provide as an extra, but that's not always what the user wants ... I mean the full list?

Reply:
Actually I think if the page size can be 1000 we never need it, so not worth it